src/parser/bounds.rs

```diff
@@ -0,0 +1,112 @@
+
+use errors::*;
+
+use std::iter::Peekable;
+use std::io::Read;
+use xml::reader::Events;
+use xml::reader::XmlEvent;
+use geo::Bbox;
+
+use GpxVersion;
+
+/// consume consumes an element as a nothing.
+pub fn consume<R: Read>(reader: &mut Peekable<Events<R>>, _version: GpxVersion) -> Result<Bbox<f64>> {
```

Comment:
Also, you should run `rust-fmt`, Travis is complaining.

Reply:
Done. This is will be in the updated PR.